paypal/paypal-messaging-components
# feat: Add blur to Calculator input

## Description

Add blur to Calculator input so on-screen keyboards close automatically in mobile web and native apps.

Renamed comboCheck to selectedAll for clarity.

## Testing instructions

1. Open a modal with a calculator input on a device that has an on-screen keyboard
2. Using the mouse, click enter (either with or without an amount)
3. The keyboard should close
